pkg/testgridanalysis/testgridconversion/to_raw_data.go

```diff
@@ -116,6 +116,54 @@ func isOverallTest(testName string) bool {
 	return testName == overall || strings.HasSuffix(testName, ".Overall")
 }
 
+// specific set of test names that include random characters in them
+// if they match the known start then look to replace the random chars with 'random'
+var matchRandomStart = regexp.MustCompile("^\"Installing \"Red Hat Integration")
+var matchRandom = regexp.MustCompile("operator in test-[a-z]+")
+var matchRandomReplace = "operator in test-random"
```

Comment:
Could you flip this to "operator in test namespace", that should be slightly more descriptive for the real test name.

Reply:
definitely. done.